### DISCUSSION TITLE
Is there a way to change IRremoteESP8266 tolerance?

### DISCUSSION THREAD

**Discussion by hr-kapanakov** [arendst/Tasmota#14435] [Q&A] [answered]
I bought an IR blaster which I flashed with tasmota-ir build and when I tried to read the signal from my two ACs remote controls I always get UNKNOWN. I verified the produced rawdata through the IRremoteESP8266 algorithm and it seems decodable, but with bigger tolerance.
So is there a way to change it through the Tasmota setting? I wasn't able to find such in the documentation.

**Comment by sfromis** [comment, marked as answer]
Don't know specifics about tolerance, but directly available tweaks are [SetOption38](https://tasmota.github.io/docs/Commands/#setoption38) and build time [options](https://github.com/arendst/Tasmota/blob/development/tasmota/my_user_config.h#L852-L857).

**Reply by hr-kapanakov** [reply]
Thanks then I'll open an issue if can be added.

**Reply by s-hadinger** [reply]
If you want more features, please submit to https://github.com/crankyoldgit/IRremoteESP8266 which is the lib used by Tasmota and where the decoding occurs.

**Reply by hr-kapanakov** [reply]
Actually, the feature is already there, but it is not exposed in Tasmota.

**Reply by sfromis** [reply]
If you post details, it may be feasible to introduce it in Tasmota. Or you could submit a PR with the change.

**Reply by hr-kapanakov** [reply]
Sure, I'll. Thank you!

**Reply by s-hadinger** [reply]
Maybe I missed it. What parameter are you referring to?

**Reply by hr-kapanakov** [reply]
I mean this one: https://crankyoldgit.github.io/IRremoteESP8266/doxygen/html/classIRrecv.html#aa091c449db70c65fd0221669df7438ea
